services/submission.js

```diff
@@ -1175,7 +1189,7 @@ class Submission {
     _verifyStudyInUserStudies(user, studyId){
         if(!user?.studies || user.studies.length === 0 )
             return false;
-        const userStudy = (user.studies[0] instanceof Object)? user.studies.find(s=>s.id === studyId || s.id === "All"):
+        const userStudy = (user.studies[0] instanceof Object)? user.studies.find(s=> s.id === studyId || s._id === studyId || s.id === "All"):
```

Comment:
Shouldn't it be  **s._id ** instead of **s.id** in last condition?

Reply:
_id and id are being used in the submission service. The user session object is currently uses id. We should update the session object in the future too.